### ISSUE TITLE
Add documentation and examples

### ISSUE THREAD

**Issue by schlenger** [keijiro/KinoMotion#26]
Right now, everything is pretty bare bone and it's hard to debug occurring errors if one is not even sure if the setup is done correctly.

**As far as I understood:**

- Integrate this package
- Add the Motion Blur post processing to the PP volume
- Add the "Motion" script to the camera

For me, this wasn't working with the new LTS version 2021.3.8f1 on MacOS with metal/silicon. It would be helpful to have at least some basic documentation in e.g. the Readme file.

**Comment by keijiro** [comment]
There is no advantage in this package compared to the standard ones. I'd recommend using them in genral. Or is there any reason to use this?

**Comment by schlenger** [comment]
@keijiro I've experienced some weird behavior and errors with the build in module. After a while, I've stumbled upon KinoMotion and I wanted to check if this solves the issues (jittering of the object the camera follows).

> There is no advantage in this package compared to the standard ones. I'd recommend using them in genral. 

I understand that the development changes and that URP didn't support MotionBlur in the PP settings at first, but maybe this line (quote) should then be added to the Readme for everyone who stumbles over this project in the future.


Side note: I love your work, you're doing an awesome job, thanks!
